scanapp/tasks.py

```diff
@@ -57,6 +63,44 @@ def scan_code_async(url, scan_id, path, file_name):
 
 
 @app.task
+def scan_code_async_final(url, scan_id, path):
+    """
+    Create and initialise the git repository at a certain path and clone the git repo using the 'url'
+    and then get the scan done asynchronously.
+    """
+    logger = logging.getLogger(__name__)
+    logger.info('git repo detected')
+
+    home_path = expanduser("~")
+    os.chdir(home_path)
```

Comment:
I am actually moving to the child directory and creating a new temp directory over there.